src/main/java/org/zalando/riptide/RestBase.java

```diff
@@ -77,4 +88,43 @@ public D execute(final HttpMethod method, final URI url, final HttpHeaders heade
 
     protected abstract <T> D execute(final HttpMethod method, final URI url, final HttpEntity<T> entity);
 
+    static <T> void writeRequestEntity(final HttpEntity<T> entity, final ClientHttpRequest request, List<HttpMessageConverter<?>> converters) throws IOException {
```

Comment:
Personally I'd prefer to set this header manually in my riptide call, but that could be a good fallback.